pkg/editing/issue.go

```diff
@@ -0,0 +1,16 @@
+// See editing/service.go for this package's comment
+package editing
+
+// This struct is a representation of a Jira issue in the context of editing it.
+type Issue struct {
+	Fields map[string]interface{} `json:"fields"`
+}
+
+// This method adds a field in the existing map of an issue before submitting this edit to the Jira API.
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [comment on exported method Issue.AddField should be of the form "AddField ..."](https://app.codacy.com/app/TurnsCoffeeIntoScripts/jira-api-resource/pullRequest?prid=4570750)